src/Main.java

```diff
@@ -1,5 +1,106 @@
+import java.util.Scanner;
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        //задача 1
+        Scanner console = new Scanner(System.in);
+        System.out.print("Введите возраст ");
+        int age = console.nextInt();
+        if (age >= 18){
+            System.out.println("он совершеннолетний");
+        } else {
+            System.out.println("он не достиг совершеннолетия");
+        }
+        //задача 2
+        System.out.print("Введите температуру ");
+        Scanner consoleTemp = new Scanner(System.in);
+        int temperature = consoleTemp.nextInt();
+        if (temperature < 5){
+            System.out.println("На улице " + temperature + " градусов(а)  нужно надеть шапку");
+        } else {
+            System.out.println("На улице " + temperature + " градусов(а) можно идти без шапки");
+        }
+        // Задача 3
+        System.out.print("Введите скорость ");
+        Scanner consoleSpeed = new Scanner(System.in);
+        int speed = consoleSpeed.nextInt();
+        if (speed >= 60){
+            System.out.println("Если скорость " + speed + "км, то придется заплатить штраф");
+        } else {
+            System.out.println("Если скорость " + speed + "км, то можно ездить спокойно");
+        }
+        // Задача 4
+        System.out.print("Введите возраст ");
+        Scanner consoleAge = new Scanner(System.in);
+        int Age = consoleAge.nextInt();
+        if (Age >= 2 && Age < 7){
+            System.out.println("Если возраст человека равен " + Age +" , то ему нужно ходить в детский сад");
+        } else {
+            if (Age >= 7 && Age < 18) {
+                System.out.println("Если возраст человека равен " + Age + " , то ему нужно ходить в школу");
+            } else {
+                if (Age >= 18 && Age < 25) {
+                    System.out.println("Если возраст человека равен " + Age + " , то его место в университете");
+                } else {
+                    System.out.println("Если возраст больше " + Age + " , то ему пора ходить на работу");
+                }
+            }
+        }
```

Comment:
Лучше воспользоваться конструкцией else if
Например:
```suggestion
        if (Age >= 2 && Age < 7){
            System.out.println("Если возраст человека равен " + Age +" , то ему нужно ходить в детский сад");
        } else if (Age >= 7 && Age < 18) {
            System.out.println("Если возраст человека равен " + Age + " , то ему нужно ходить в школу");
        } else if (...) {
           //и т.д.
        }
```